run-tests.php

```diff
@@ -3297,6 +3301,35 @@ function show_result(
         } else {
             echo "$result $tested [$tested_file] $extra\n";
         }
+
+        if ($github_actions && $result !== 'PASS') {
+            $sanitized_title = preg_replace('/([\\r\\n]|::)/', '', $tested);
```

Comment:
In case you are interessted in proper escaping, see https://github.com/staabm/annotate-pull-request-from-checkstyle/blob/69c6493d155c2b745619a13fc521b097f9fd1acd/cs2pr#L136 for inspiration